### PR TITLE
oracle: replace pingcap/check with testify

### LINKED ISSUE #118: Replace check with testify: /oracle/oracles
subtask of https://github.com/tikv/client-go/issues/97

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

closes #118 

cc @disksing

### CODE REVIEW

**Comment by tisonkun** [comment]
closed as taken by @disksing

File: oracle/oracles/local_test.go [hunk 1]
```diff
@@ -34,49 +34,52 @@ package oracles_test
...
 	m := map[uint64]struct{}{}
 	for i := 0; i < 100000; i++ {
 		ts, err := l.GetTimestamp(context.Background(), &oracle.Option{})
-		c.Assert(err, IsNil)
+		require.Nil(t, err)
 		m[ts] = struct{}{}
 	}
 
-	c.Assert(len(m), Equals, 100000, Commentf("should generate same ts"))
+	assert.Len(t, m, 100000)
```
Comment (disksing) [inline oracle/oracles/local_test.go:56]: it would better to keep comment (add the comment string as an extra argument)

Comment (tisonkun) [reply oracle/oracles/local_test.go:56]:
```suggestion
	assert.Len(t, m, 100000, "should generate same ts")
```

Comment (tisonkun) [reply oracle/oracles/local_test.go]: Updated.

File: oracle/oracles/local_test.go [hunk 2]
```diff
@@ -34,49 +34,52 @@ package oracles_test
...
 	defer o.Close()
+
 	start := time.Now()
 	oracles.SetOracleHookCurrentTime(o, start)
 	ts, _ := o.GetTimestamp(context.Background(), &oracle.Option{})
 	oracles.SetOracleHookCurrentTime(o, start.Add(10*time.Millisecond))
+
 	expire := o.IsExpired(ts, 5, &oracle.Option{})
-	c.Assert(expire, IsTrue, Commentf("should expire"))
+	assert.True(t, expire)
```
Comment (disksing) [inline oracle/oracles/local_test.go:69]: better to keep comment here too.

File: oracle/oracles/local_test.go [hunk 3]
```diff
@@ -34,49 +34,52 @@ package oracles_test
...
+
 	expire = o.IsExpired(ts, 200, &oracle.Option{})
-	c.Assert(expire, IsFalse, Commentf("should not expire"))
+	assert.False(t, expire)
```
Comment (disksing) [inline oracle/oracles/local_test.go:72]: and here.

File: oracle/oracles/pd_test.go [hunk 4]
```diff
@@ -38,70 +38,35 @@ import (
...
+
 	start := time.Now()
 	oracles.SetEmptyPDOracleLastTs(o, oracle.GoTimeToTS(start))
 	ts, err := o.GetStaleTimestamp(context.Background(), oracle.GlobalTxnScope, 10)
-	c.Assert(err, IsNil)
-
-	duration := start.Sub(oracle.GetTimeFromTS(ts))
-	c.Assert(duration <= 12*time.Second && duration >= 8*time.Second, IsTrue, Commentf("stable TS have accuracy err, expect: %d +-2, obtain: %d", 10, duration))
+	assert.Nil(t, err)
+	assert.WithinDuration(t, start.Add(-10*time.Second), oracle.GetTimeFromTS(ts), 2*time.Second)
```
Comment (disksing) [inline oracle/oracles/pd_test.go:63]: here too.

File: oracle/oracles/local_test.go (hunk 3 above)
Comment (tisonkun) [reply oracle/oracles/local_test.go:72]: I think it is redundant comment.

File: oracle/oracles/local_test.go (hunk 2 above)
Comment (tisonkun) [reply oracle/oracles/local_test.go:69]: I think it is redundant comment.

File: oracle/oracles/pd_test.go (hunk 4 above)
Comment (tisonkun) [reply oracle/oracles/pd_test.go:63]: It is a workaround to display more info then True/False, with `withinDuration` we don't meet this issue.

File: oracle/oracles/local_test.go (hunk 3 above)
Comment (disksing) [reply oracle/oracles/local_test.go:72]: good point.

**Review by disksing — APPROVED** [review]

**Pull request merged**